modules/nf-core/geofetch/meta.yml

```diff
@@ -0,0 +1,36 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "geofetch"
+description: geofetch is a command-line tool that downloads and organizes data and metadata from GEO and SRA
+keywords:
+  - GEO
+  - expression
+  - microarray
+  - sequencing
+tools:
+  - "geofetch":
+      description: "Downloads data and metadata from GEO and SRA and creates standard PEPs."
+      homepage: "http://geofetch.databio.org/"
+      documentation: "http://geofetch.databio.org/"
+      tool_dev_url: "https://github.com/pepkit/geofetch"
+      licence: ["BSD-2-clause"]
+
+input:
+  - geo_accession:
+      type: string
+      description: GEO accession ID
+
+output:
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - samples:
```

Comment:
Shouldn't this be two entries? (see point 7 in https://nf-co.re/docs/contributing/modules#documentation)

Reply:
True!